.github/workflows/validate_template.yml

```diff
@@ -0,0 +1,25 @@
+name: Validate Template has an issue in a description
+
+on:
+  pull_request:
+    types: [edited, opened]
+
+jobs:
+  herald:
+    name: Validate Template
+    runs-on: ubuntu-latest
+    steps:
+      - name: event location
+        run: |
+          echo $GITHUB_EVENT_PATH
+      - uses: actions/upload-artifact@master
+        with:
+          name: event
+          path: /home/runner/work/_temp/_github_workflow/event.json
```

Comment:
what does this do?
the absolute path into a temp directory makes me a little nervous

Reply:
yea, I tried to use the actual path from a variable but it didn't catch it, have you tried it ?

Reply:
I have not tried it before 😬

Reply:
`github.event_path`!